code/modules/antagonists/brainwashing/brainwashing.dm

```diff
@@ -22,8 +22,9 @@
 	var/end_message = "."
 	var/rendered = begin_message + obj_message + end_message
 	deadchat_broadcast(rendered, "<b>[L]</b>", follow_target = L, turf_target = get_turf(L), message_type=DEADCHAT_ANNOUNCEMENT)
-	if(prob(1) || check_holidays(APRIL_FOOLS))
-		L.say("You son of a bitch! I'm in.", forced = "That son of a bitch! They're in.")
+	if(check_holidays(APRIL_FOOLS))
```

Comment:
Maybe instead we should have it exponentially get more likely the more people get converted, before resetting? Not a blocking comment but it seems better to me than doing it for everyone

Reply:
Oh wait this is brainwashing, carry on